src/cr/cube/subtotal.py

```diff
@@ -43,10 +46,15 @@ def anchor(self):
     def _all_dim_ids(self):
         return [el.get('id') for el in self._dim.elements(include_missing=True)]
 
+    def _validate_hs_ids(self, hs_ids):
+        element_ids = [el['id'] for el in self._dim.elements()]
+        return any(arg in element_ids for arg in hs_ids)
```

Comment:
Why not just: `[el for el in self._dim.elements() if el['id'] in hs_ids]`?

Why not just inline this in `is_valid`? It would make `is_valid` easier to read.

(If your answer would be, it's used in `args` below, see my comment there. :)